app/src/controllers/dashboard/selectors.js

```diff
@@ -40,4 +41,6 @@ export const totalDashboardsSelector = (state) =>
 
 export const dashboardFullScreenModeSelector = (state) => domainSelector(state).fullScreenMode;
 
-export const querySelector = createQueryParametersSelector();
```

Comment:
Do we need to revert the changes from https://jiraeu.epam.com/browse/EPMRPP-81029 or adjust them somehow?